### ISSUE TITLE
how to change states on devices using pyvivint

### ISSUE THREAD

**Issue by swincher4391** [ovirs/pyvivint#11]
I looked at demo.py and was able to print out my devices... cool.  Now how do I use mutator functions to do things like bypass a wireless sensor?

**Comment by natekspencer** [comment]
Sensors currently don't have a function to do that. I don't ever bypass my sensors manually, so it never came up to code a function to do so.

It's an easy addition though. Just a PUT request to `{{base_url}}/{{panel_id}}/{{partition_id}}/sensors/{{device_id}}` with:
```json
{
  "b": <bypass_value>
}
```
where bypass_value is 0 for unbypassed, 1 for force bypassed and 2 for manually bypassed.

I'll get around to it later unless you want to take care of this one @ovirs?

**Comment by swincher4391** [comment]
I would do that in vivintskyapi?

**Comment by natekspencer** [comment]
Yeah, you could copy the set_lock_state function and modify it for the sensor command

**Comment by swincher4391** [comment]
I’ll give it a go, thank you.
